es6-promises/index.js

```diff
@@ -0,0 +1,8 @@
+const takeAChance = require('./take-a-chance');
+const returnValue = takeAChance('Garrett');
+returnValue.then(value => {
+  console.log(value);
+});
+returnValue.catch(error => {
+  console.error(error);
+});
```

Comment:
Your rejection handler should only output the error message, not the whole error object:

---

![image](https://user-images.githubusercontent.com/16143771/180913121-fefc64b7-c016-4a33-b618-23ce24aee8df.png)

---

![image](https://user-images.githubusercontent.com/16143771/180913142-1bd33792-5dec-479c-97a8-1afb4529ded0.png)